apps/admin-ui/src/component/reservations/Filters.tsx

```diff
@@ -132,34 +153,53 @@ export function Filters({
     <Wrapper>
       <AutoGrid>
         <MultiSelectFilter
-          options={reservationUnitTypeOptions}
-          name="reservationUnitType"
+          options={reservationTypeOptions}
+          name="reservationType"
         />
         <MultiSelectFilter options={stateOptions} name="state" />
-        <MultiSelectFilter options={unitOptions} name="unit" />
-        <SearchFilter name="search" labelKey="searchReservation" />
-        <MultiSelectFilter
-          options={paymentStatusOptions}
-          name="paymentStatus"
-        />
         <MultiSelectFilter
           options={reservationUnitOptions}
           name="reservationUnit"
         />
-        <DateInputFilter name="begin" />
-        <DateInputFilter name="end" />
+        <SearchFilter name="search" labelKey="searchReservation" />
       </AutoGrid>
       <MoreWrapper
         showAllLabel={t("ReservationUnitsSearch.moreFilters")}
         showLessLabel={t("ReservationUnitsSearch.lessFilters")}
         maximumNumber={0}
       >
         <AutoGrid>
+          <DateRangeFilter name="date" />
+          <MultiSelectFilter options={unitOptions} name="unit" />
+          <MultiSelectFilter
+            options={reservationUnitTypeOptions}
+            name="reservationUnitType"
+          />
           <RangeNumberFilter
             label={t("filters.label.price")}
             minName="minPrice"
             maxName="maxPrice"
           />
+          <MultiSelectFilter
+            name="orderStatus"
+            options={paymentStatusOptions}
+          />
+          <DateRangeFilter name="createdAt" />
+          <Select
+            label={t("filters.label.isRecurring")}
+            id="isRecurring"
+            options={recurringOptions}
+            onChange={(val) => {
+              const params = new URLSearchParams(searchParams);
+              if (val != null) {
+                params.set("recurring", val.toString());
+              } else params.delete("recurring");
+              setParams(params, { replace: true });
+            }}
+            value={new URLSearchParams(searchParams).get("recurring") ?? ""}
+            clearable
+          />
```

Comment:
Separate component similar to others.

Reply:
Good point! Extracted to it's own component (`SelectFilter.tsx`)